Hudl.FFmpeg.Core/Command/BaseTypes/ICommand.cs

```diff
@@ -12,7 +12,11 @@ public interface ICommand
             where TProcessorType : class, ICommandProcessor, new()
             where TBuilderType : class, ICommandBuilder, new();
 
-        ICommandProcessor ExecuteWith<TProcessorType, TBuilderType>(TProcessorType commandProcessor)
+        ICommandProcessor ExecuteWith<TProcessorType, TBuilderType>(int? timeoutMilliseconds)
+            where TProcessorType : class, ICommandProcessor, new()
+            where TBuilderType : class, ICommandBuilder, new();
+
+        ICommandProcessor ExecuteWith<TProcessorType, TBuilderType>(TProcessorType commandProcessor, int? timeoutMilliseconds)
```

Comment:
@rversaw Yeah, I changed this up just recently to add these timeouts to new signatures. so that it doesn't break existing code.